libmikmod/playercode/mplayer.c

```diff
@@ -2806,30 +2909,32 @@ static void pt_UpdateVoices(MODULE *mod, int max_volume)
 		envpan = PAN_CENTER;
 		envpit = 32;
 		if (i && ((aout->main.kick==KICK_NOTE)||(aout->main.kick==KICK_ENV))) {
-			if (aout->main.volflg & EF_ON)
-				envvol = StartEnvelope(&aout->venv,aout->main.volflg,
-				  i->volpts,i->volsusbeg,i->volsusend,
-				  i->volbeg,i->volend,i->volenv,aout->main.keyoff);
-			if (aout->main.panflg & EF_ON)
-				envpan = StartEnvelope(&aout->penv,aout->main.panflg,
-				  i->panpts,i->pansusbeg,i->pansusend,
-				  i->panbeg,i->panend,i->panenv,aout->main.keyoff);
+			if (aout->main.volflg & EF_ON) {
```

Comment:
The changes in this section look 100% correct, I would merely recommend staying close to the old text wrapping where possible (I really hope no one relies on 96-char wrapping though...).

Reply:
Never mind, the text wrapping in other parts of this file are already a mess.